fastlane_bot/events/exchanges/solidly_v2.py

```diff
@@ -41,31 +60,23 @@ def add_pool(self, pool: Pool):
         self.pools[pool.state["address"]] = pool
 
     def get_abi(self):
-        return VELOCIMETER_V2_POOL_ABI if "velocimeter" in self.exchange_name else SOLIDLY_V2_POOL_ABI
+        return EXCHANGE_INFO.get(self.exchange_name).get("pool_abi")
```

Comment:
You can use `[]` instead of `get`, it is cleaner and more readable.

Reply:
updated